Userland/Libraries/LibCore/Process.h

```diff
@@ -151,4 +151,7 @@ class IPCProcess {
     Process m_process;
 };
 
+ErrorOr<Optional<pid_t>> get_process_pid(StringView process_name, StringView pid_path);
+ErrorOr<int> create_ipc_socket(ByteString const& socket_path);
```

Comment:
nit: It might be nicer for these to be `static public` on the `IPCProcess` class. Just feels a little weird having some of its mechanics be that way, and then these being free functions